tests/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(32, 6)) == 32
 
 
 def test_should_split_a_number_into_equal_parts_when_value_is_divisible_by_number_of_parts():
-    pass
+    assert len(split_integer(25, 5)) == 5
```

Comment:
I think, this testcase should check if all parts are equal, and not if there amount is equal to number_of_parts. Don't use len, just check if split_integer(25, 5) gives a correct array.

Reply:
thx) I fixed it.